### ISSUE TITLE
[Settings]: Screen Reader is not announcing respective group label for all 'Reset to inherited value' button.

### ISSUE THREAD

**Issue by ghost** [microsoft/terminal#12016]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1741.1

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Open Narrator using 'Win+Ctrl+ Enter' key.
4. Navigate to 'Reset to inherited value' controls throughout the page and observe the issue.

**User Experience:**
Screen Reader users UX will not be good if Screen Reader doesn't announce group label with control as result users will not able to know the purpose of control.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

**Attachments:**
[Screen Reader is not announcing group label for 'Reset to inherited value' button..zip](https://github.com/microsoft/terminal/files/7762291/Screen.Reader.is.not.announcing.group.label.for.Reset.to.inherited.value.button.zip)

### Expected Behavior

Screen Reader should announce respective group label with all 'Reset to inherited value' button.
i.e. For 'Reset to inherited value' button present beside 'Command line' edit field, Screen Reader should announce 'Command line group Reset to inherited value' button.

### Actual Behavior

Screen Reader is not announcing respective group label for all 'Reset to inherited value' button.
i.e. For 'Reset to inherited value' button present beside 'Command line' edit field, Screen Reader is only announcing 'Reset to inherited value' button.

**Observation:** Same issue repro with NVDA and JAWS Screen Reader.

**Comment by Priyanka-Chauhan123** [comment]
Closing the bug as UI has been changed.
